vertx-config/src/main/java/io/vertx/config/ConfigStream.java

```diff
@@ -0,0 +1,22 @@
+package io.vertx.config;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.streams.ReadStream;
+
+/**
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+@VertxGen
+public interface ConfigStream extends ReadStream<JsonObject> {
```

Comment:
now we don't need this anymore as `ConfigRetrieve#configStream()` can return `ReadStream<JsonObject>`